README.fa.md

```
@@ -0,0 +1,120 @@
+<p align="center" style="text-align: center">
+  <img src="./assets/images/logo-universal.png" width="55%"><br/>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)